### PR TITLE
doc: remove @bnb from moderation

### PR DESCRIPTION
**Pull request by bnb**
Disappointed in how this was handled.

### CODE REVIEW

**Comment by bnb** [comment]
cc @nodejs/tsc @nodejs/moderation

File: Moderation-Policy.md [hunk 1]
```diff
@@ -294,6 +292,8 @@ remove resigning team member from respective permissions and private access.
 **Alexander Tesfamichael** &lt;alex.tesfamichael@gmail.com&gt;
 * [aredridel](https://github.com/aredridel) -
 **Aria Stewart** &lt;aredridel@dinhe.net&gt;
+* [bnb](https://github.com/bnb) -
```
Comment (RaisinTen) [inline Moderation-Policy.md:295]: This addition has been made to the `### Admins for Node.js Slack community` section, is that intentional? Looking at https://github.com/nodejs/admin/pull/254, it seems that the only change should be the removal of lines 276 and 277 that you have done already at the top.

Comment (bnb) [reply Moderation-Policy.md]: my bad, I just saw the list and assumed it was emeritus without actually reading.

**Review by RaisinTen — APPROVED** [review]
LGTM, thanks

**Review by aduh95 — APPROVED** [review]

**Review by richardlau — APPROVED** [review]

**Review by gireeshpunathil — APPROVED** [review]

**Review by Trott — APPROVED** [review]

**Comment by Trott** [comment]
Thank you for your service and contributions doing your tenure on the Moderation Team.

**Pull request merged**